config/initializers/utility.rb

```diff
@@ -0,0 +1,3 @@
+def time_parser(time_data)
+  time_data.to_time.strftime("%Y-%m-%d %H:%M:%S")
```

Comment:
Do not use `to_time` on Date objects, because they know nothing about the time zone in use.